cluster-state-service/internal/features/e2etasksteps/common_steps.go

```diff
@@ -34,48 +38,113 @@ var (
 	cssTaskList   = []models.Task{}
 	exceptionList = []Exception{}
 
-	taskDefnARN = ""
+	taskDefnARN         = ""
+	roleName            = "E2ETestRole"
+	instanceProfileName = "E2ETestInstance"
+	policyARN           = "arn:aws:iam::aws:policy/service-role/AmazonEC2ContainerServiceforEC2Role"
 )
 
+const errorCode = 1
+
 func init() {
 
 	cssWrapper := wrappers.NewCSSWrapper()
 	ecsWrapper := wrappers.NewECSWrapper()
+	ec2Wrapper := wrappers.NewEC2Wrapper()
+	iamWrapper := wrappers.NewIAMWrapper()
 
 	BeforeAll(func() {
-		var err error
+		clusterName := wrappers.GetClusterName()
```

Comment:
Can you add a comment above `BeforeAll` that says why you're using `os.Exit` here instead of `T.Errorf`? Maybe something like:

// TODO: Change these os.Exit calls to T.Errorf. Currently unable to do so because T is not initialized until the first test.
// (https://github.com/gucumber/gucumber/issues/28)